src/server_manager/install_scripts/install_server.sh

```diff
@@ -29,11 +27,23 @@
 #     only by do_install_server.sh.
 # WATCHTOWER_REFRESH_SECONDS: refresh interval in seconds to check for updates,
 #     defaults to 3600.
+#
+# Deprecated:
+# SB_PUBLIC_IP: Use the --hostname flag instead
+# SB_API_PORT: Use the --port-for-api flag instead
 
 # Requires curl and docker to be installed
 
 set -euo pipefail
 
+function display_usage() {
+  echo Usage: install_server.sh [--hostname <hostname>] [--port-for-api <port>] [--port-for-keys <port>]
+  echo
+  echo  --hostname        The hostname to be used to access the management API and access keys
+  echo  --port-for-api    The port number for the management API
```

Comment:
Consider renaming to `--api-port`, `--keys-port`. Same with the `FLAGS` variables.